project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject hiposfer/kamal "0.1.1-SNAPSHOT"
+(defproject hiposfer/kamal "0.1.3"
```

Comment:
@carocad I think removing `SNAPSHOT` has broken deploy to clojars.

Reply:
@mehdisadeghi I thought something similar. What do you think of the following workflow.

We would have two branches: `dev` and `master`. Everything that is published to `dev` must be with the `snapshot` version, whereas everything that goes into `master` would be `release` ready. That way we can keep both deploy scripts :)

What do you think?

Reply:
@carocad I think this is unnecessary. IMO code on Github is always development version. Keeping only one branch, namely master, makes everything simpler. We have to solve this problem properly, it is not urgent though.

Reply:
After more detailed investigation I think the failure to deploy has nothing to do with the missing `SNAPSHOT`. Most probably clojars deploy step was not executed at all, because the Github deploy step has produced an error (file not found) and has caused the deploy script to die.